tests/rptest/tests/random_node_operations_test.py

```diff
@@ -58,6 +58,8 @@ def __init__(self, test_context, *args, **kwargs):
                 # set disk timeout to value greater than max suspend time
                 # not to emit spurious errors
                 "raft_io_timeout_ms": 20000,
+                "compacted_log_segment_size": 1024 * 1024,
+                "log_segment_size": 2 * 1024 * 1024,
```

Comment:
just fyi, we can go smaller than this with segment size using https://github.com/redpanda-data/redpanda/pull/24544

`environment = {"__REDPANDA_TEST_DISABLE_BOUNDED_PROPERTY_CHECKS": "ON"}`